src/pages/games/terraforming-mars/Match.tsx

```diff
@@ -22,7 +22,7 @@ import { useRouteMatch } from "react-router-dom";
 import theme from "../../../theme";
 import { Rating } from "@material-ui/lab";
 
-const metrics = ["achievements", "awards", "cards", "cities", "forests"];
+const metrics = ["rating", "achievements", "awards", "cards", "cities", "forests"];
```

Comment:
```suggestion
const metrics = ["terraform rating", "achievements", "awards", "cards", "cities", "forests"];
```
This also is more than default line length for prettier, run `prettier:fix`